sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/AllBranchesShouldNotHaveSameImplementation.cs

```diff
@@ -145,22 +145,20 @@ private void DoSomethingElse()
         {
         }
 
-        public int SwitchExpressionNoncompliant(string type)
-        {
-            return type switch // Compliant - FN: all branches call the same function
+        public int SwitchExpressionNoncompliant(string type) =>
+            type switch // Noncompliant
```

Comment:
It would be nice to have a check on the message and location for the switch expression example too